.travis.yml

```diff
@@ -7,6 +7,7 @@ node_js:
 services:
   - postgresql
   - redis-server
+  - docker
```

Comment:
@bhaskaryasa 
The above changes don't appear to be yours but somehow showed up in your push.

Looks like you are pulling in changes from some other branch 
or
you might have missed a rebase step

I see there are some other similar changes.